app/shared/types.js

```diff
@@ -23,3 +23,11 @@ export type ReadabilityResult = {
   length: number,
   excerpt: string,
 };
+
+export type AwesomebarMatch = {
```

Comment:
I feel this pattern is not helping.  Maybe we can just use ES6 class definitions for this?